src/climate_assessment/climate/ciceroscm.py

```diff
@@ -19,7 +19,7 @@ def get_ciceroscm_configurations(
         # version strings for linux and windows might be different!
         raise AssertionError(CICEROSCM.get_version())
 
-    with open(ciceroscm_probabilistic_file, "r") as fh:
+    with open(ciceroscm_probabilistic_file) as fh:
```

Comment:
Why is the "r" removed?

Reply:
Ruff removes it. I think because "r" is default you don't need to write it.